helix-core/src/syntax.rs

```diff
@@ -519,25 +518,33 @@ impl Loader {
 
     pub fn language_configuration_for_injection_string(
```

Comment:
This function can be cleaned up a bit if we move the code for `InjectionCapture::Name` to its own function `language_config_for_name`

Reply:
That's right. I meant to do that but I guess I overlooked it. Just force pushed a fix.